client/jetpack-connect/plans.jsx

```diff
@@ -223,6 +232,7 @@ export default connect(
 			isRtlLayout: isRtl( state ),
 			hasPlan: selectedSite ? isCurrentPlanPaid( state, selectedSite.ID ) : null,
 			notJetpack: selectedSite ? ! isJetpackSite( state, selectedSite.ID ) : null,
+			isSitesInitialized: hasInitializedSites( state ),
```

Comment:
Whereas `isRequestingSites` returns false before the site data is actually available, `hasInitializedSites` will not return true until the site data is present.